Content/Afflictions.xml

```diff
@@ -2756,4 +2756,24 @@
     <icon texture="Content/UI/CommandUIAtlas.png" sourcerect="768,768,128,128" color="143,111,160,255" origin="0,0"/>
   </Affliction>
 
+  <Affliction
+    name="BoxSlowdown"
+    identifier="ammoboxslowdown"
+    description=""
+    showiconthreshold="1000"
+    showicontoothersthreshold="1000"
+    showinhealthscannerthreshold="1000"
+    type="slowness" 
+    limbspecific="false"
+    indicatorlimb="Torso"
+    maxstrength="2"
+    healableinmedicalclinic="false">
+    <Effect minstrength="0" maxstrength="1" strengthchange="-1">
+      <StatusEffect type="OnActive" target="Character" SpeedMultiplier="0.8" PropulsionSpeedMultiplier="0.8" setvalue="true" duration="0.1" />
+    </Effect>
+    <Effect minstrength="1" maxstrength="2" strengthchange="-2">
+      <StatusEffect type="OnActive" target="Character" SpeedMultiplier="0.6" PropulsionSpeedMultiplier="0.6" setvalue="true" duration="0.1" />
+    </Effect>
+  </Affliction>

```

Comment:
It's not clear to me why this is implemented using an affliction, couldn't the effects be in the items themselves?

Reply:
Because the slowdown effects don't stack, Statuseffects will use setvalue

From moloch shield:

`      <StatusEffect type="OnActive" target="Character" SpeedMultiplier="0.6" setvalue="true" />`

Reply:
Ah, I see! I think a cleaner solution would be to use stat values for this, because they do stack. Those only seem to be supported on Wearables though, but seems like it'd be easy to add support to holdable items as well.

Reply:
Yes, switching to stat values if they become functional in holdable components would definitely be the better solution.